interfaces/builtin/pulseaudio.go

```diff
@@ -20,30 +20,170 @@
 package builtin
 
 import (
+	"bytes"
+
 	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/release"
 )
 
-const pulseaudioConnectedPlugAppArmor = `
+var pulseaudioConnectedPlugAppArmor = []byte(`
+capability dac_override,
```

Comment:
Please remove this. From @morphis "But to be honest, I can't remember right now why this was required for the plug side as setuid/setgid happens on the slot side." The connected plug should not need elevated privileges to work with pulseaudio, which this capability rule grants. If after removing it you still need it, please investigate way and adjust the slot snap to not require the plug snap to need this.